src/Engine/GameObject/GameObject.h

```diff
@@ -19,8 +22,10 @@ class GameObject : public I_Updatable, public Observable<DestroyedListener> {
     virtual void draw(I_GraphicsHandler& graphics) = 0;
     virtual const I_Hitbox& hitbox() const = 0;
     virtual void notify_collision(GameObject& collided_object) = 0;
+
     virtual void take_damage(int damage);
 
+    std::string get_sound();
```

Comment:
This used anywhere?

Reply:
no. I think I had plans for this before but not anymore